<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="search_line">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="search_btn">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.352, x2:0, y2:0, stop:0.165854 rgba(240, 240, 240, 255), stop:0.517073 rgba(244, 244, 244, 255), stop:0.731707 rgba(244, 244, 244, 255), stop:0.936585 rgba(243, 243, 243, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../UI/magnifying-glass.png</normaloff>../UI/magnifying-glass.png</iconset>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>1051</width>
     <height>781</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;KoPub돋움체 Bold&quot;;</string>
   </property>
   <property name="rowCount">
    <number>20</number>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
    <property name="font">
     <font>
      <family>KoPub돋움체 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
    <property name="font">
     <font>
      <family>KoPub돋움체 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </row>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Serial_no</string>
    </property>
    <property name="font">
     <font>
      <family>KoPub돋움체 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Img_path_1</string>
    </property>
    <property name="font">
     <font>
      <family>KoPub돋움체 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Img_path_2</string>
    </property>
    <property name="font">
     <font>
      <family>KoPub돋움체 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Result</string>
    </property>
    <property name="font">
     <font>
      <family>KoPub돋움체 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Img1_score</string>
    </property>
    <property name="font">
     <font>
      <family>KoPub돋움체 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Img2_score</string>
    </property>
    <property name="font">
     <font>
      <family>KoPub돋움체 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Check_date</string>
    </property>
    <property name="font">
     <font>
      <family>KoPub돋움체 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="previous_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>7</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;KoPub돋움체 Bold&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0.352, x2:0, y2:0, stop:0.165854 rgba(240, 240, 240, 255), stop:0.517073 rgba(244, 244, 244, 255), stop:0.731707 rgba(244, 244, 244, 255), stop:0.936585 rgba(243, 243, 243, 255));</string>
   </property>
   <property name="text">
    <string>이전</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_btn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>7</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;KoPub돋움체 Bold&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0.352, x2:0, y2:0, stop:0.165854 rgba(240, 240, 240, 255), stop:0.517073 rgba(244, 244, 244, 255), stop:0.731707 rgba(244, 244, 244, 255), stop:0.936585 rgba(243, 243, 243, 255));</string>
   </property>
   <property name="text">
    <string>다음</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
